### PR TITLE
Null check for getSupportActionBar()

### LINKED ISSUE #624: Crash: Unable to start welcome activity on fresh install
```
java.lang.RuntimeException: Unable to start activity ComponentInfo{fr.free.nrw.commons/fr.free.nrw.commons.WelcomeActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.support.v7.app.ActionBar.hide()' on a null object reference
                                                                       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
                                                                       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
                                                                       at android.app.ActivityThread.-wrap11(ActivityThread.java)
                                                                       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
                                                                       at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                       at android.os.Looper.loop(Looper.java:148)
                                                                       at android.app.ActivityThread.main(ActivityThread.java:5417)
                                                                       at java.lang.reflect.Method.invoke(Native Method)
                                                                       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
```

### PR DESCRIPTION
**Pull request by whym**
Fixes #624

### CODE REVIEW

**Comment by misaochan** [comment]
@whym sorry for not notifying, but we are planning to remove the action bar entirely once the nav drawer has been tidied up (today or tomorrow, hopefully). So maybe we will leave this for now? Thanks for submitting the fix though.

**Comment by misaochan** [comment]
Edit: Ah, I got confused. What @maskaravivek meant was that the `.hide() `line is not needed any longer.

File: app/src/main/java/fr/free/nrw/commons/WelcomeActivity.java [hunk 1]
```diff
@@ -20,7 +20,9 @@ public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_welcome);
 
-        getSupportActionBar().hide();
+        if (getSupportActionBar() != null) {
+            getSupportActionBar().hide();
```
Comment (maskaravivek) [inline app/src/main/java/fr/free/nrw/commons/WelcomeActivity.java]: We no longer need the null check as the app now uses the `NoActionBar` theme. `getSupportActionBar().hide();` can be completely removed as the activity has no action bar. Other activities have a custom action bar.

**Comment by whym** [comment]
Ah I see. Will close this.

**Pull request closed without merging**